roles/keystone/tasks/keystone_juno.yml

```diff
@@ -0,0 +1,249 @@
+---
+- name: install keystone components
+  yum: pkg={{item}} state=present
+  with_items:
+    - openstack-keystone
+    - openstack-utils
+    - httpd
+  tags: keystone
+
+- name: set keystone configuration
+  command: openstack-config {{ item }}
+  with_items:
+    - " --set /etc/keystone/keystone.conf DEFAULT admin_token {{keystone_admin_token}}"
```

Comment:
The openstack-config handles a lot of that. It  looks at the ini file and if it needs to change the line it does. If it doesnt exist it adds it. 

Sent from my iPhone

> On Jun 9, 2015, at 2:50 PM, Joseph Callen <notifications@github.com> wrote:
> 
> In roles/keystone/tasks/keystone_juno.yml:
> 
> > @@ -0,0 +1,249 @@
> > +---
> > +- name: install keystone components
> > +  yum: pkg={{item}} state=present
> > +  with_items:
> > +    - openstack-keystone
> > +    - openstack-utils
> > +    - httpd
> > +  tags: keystone
> > +
> > +- name: set keystone configuration
> > +  command: openstack-config {{ item }}
> > +  with_items:
> > +    - " --set /etc/keystone/keystone.conf DEFAULT admin_token {{keystone_admin_token}}"
> This won't be idempotent if --verbose and changed_when isn't used. Do we care at this point? Or v2?
> 
> —
> Reply to this email directly or view it on GitHub.
>